client-metrics/src/main/scala/org/http4s/client/metrics/core/MetricsOps.scala

```diff
@@ -0,0 +1,75 @@
+package org.http4s.client.metrics.core
+
+import cats.effect.Sync
+import org.http4s.Status
+
+/**
+  * Describes an algebra capable of writing metrics to a metrics registry
+  *
+  */
+trait MetricsOps[F[_]] {
+
+  /**
+    * Increases the count of active requests
+    *
+    * @param classifier the request classifier
+    */
+  def increaseActiveRequests(classifier: Option[String]): F[Unit]
```

Comment:
Should these classifiers get a newtype, or are we happy with Strings? :thinking:

Reply:
Yeah, just to avoid the problem of Everything Is a String.  Maybe it's more annoying than anything.  I could go either way.  I'll trust your judgement.

Reply:
I see your point. But it doesn't look very problematic to me in this particular case, so I would be more inclined to keep it simple and leave like it is if that's fine with you.